package.json

```diff
@@ -10,21 +10,19 @@
         "production": "mix --production"
     },
     "devDependencies": {
-        "axios": "^0.21",
+        "axios": "^0.25",
```

Comment:
Laravel 9.x package.json requires "axios": "^0.27"

Reply:
Resolved in [8cf72d2](https://github.com/ZTL-ARTCC/Current_Website/pull/176/commits/8cf72d20f1657c698b63dd99f30a2079b2f721ef)